lib/src/authorization_code_grant.dart

```diff
@@ -136,11 +139,13 @@ class AuthorizationCodeGrant {
       bool basicAuth = true,
       http.Client httpClient,
       CredentialsRefreshedCallback onCredentialsRefreshed,
+      Map<String, dynamic> additionalBody,
```

Comment:
Maybe update the documentation comment..

Reply:
added

Reply:
Document what this is, how it's used, for what... when... what request it's sent with and how..

Also it's not clear that this can be `Map<String,dynamic>`, from what I can see the body is encoded as `application/x-www-form-urlencoded`.

This should probably also be called something like `vendorParameters` or something like that... or `vendorSpecificParams`, not sure :)